feature-libs/checkout/b2b/components/checkout-payment-type/checkout-payment-type.component.spec.ts

```diff
@@ -31,6 +31,19 @@ class MockCheckoutPaymentTypeService
   );
 }
 
+class MockCheckoutOnePaymentTypeService
+  implements Partial<CheckoutPaymentTypeFacade>
+{
+  getPaymentTypes = createSpy().and.returnValue(of(mockDisableOnePaymentTypes));
+  setPaymentType = createSpy().and.returnValue(of('setPaymentType'));
```

Comment:
They type is listed as `unknown`, please use a more neutral value or a reasonable mock value for the default return.

```suggestion
  setPaymentType = createSpy().and.returnValue(of(undefined));
```

Reply:
Done.